plugins/google/client.js

```diff
@@ -20,9 +22,8 @@ function onGoogleSignIn(googleUser) {
 			}	
 		}
 	};
-	xhttp.setRequestHeader("Content-type", "application/json");
-	xhttp.setRequestHeader("Connection", "close");
 	xhttp.open("POST", "/auth/google/", true);
+	xhttp.setRequestHeader("Content-type", "application/json");
```

Comment:
better browser compatibility, since some browsers do not allow `setRequestHeader( key, value )` before `open( method, url )`